src/App.tsx

```diff
@@ -7,8 +6,13 @@ function TaskShow() {
   return (
     <div>
       {taskList.map((task) => (
-        <div>
-          <span> {task.name} </span>
+        <div className="row">
+          <div className="col1">
+            <div className="box1">{task.id}</div>
+          </div>
+          <div className="col2">
+            <div className="box2">{task.name}</div>
+          </div>
```

Comment:
If you use `flex` then there is no need to wrap each box around a `col` like this, which is much simpler and easier to manage later on

Also, I think it's better to rename the `className` to something more meaningful. For example, instead of `box1`, we can name it `it-container` and instead of `box2`, we can name it `task-name-container`